katello-ssl-tool

```diff
@@ -29,23 +29,18 @@
 
 ## language imports
 import sys
-try:
-    import os
-    import string
-except KeyboardInterrupt:
-    sys.stderr.write("\nUser interrupted process.\n")
-    sys.exit(0)
+import os
 
 
 sys.path.append("@@ROOT@@")
```

Comment:
I don't really think this is needed anymore. We never replace this with a sensible value